contentcuration/contentcuration/utils/publish.py

```diff
@@ -431,9 +433,9 @@ def create_perseus_zip(ccnode, exercise_data, write_to_path):
                                 content = content.split(exercises.GRAPHIE_DELIMITER.encode('ascii'))
                                 write_to_zipfile(svg_name, content[0], zf)
                                 write_to_zipfile(json_name, content[1], zf)
-                    write_assessment_item(question, zf)
+                    write_assessment_item(question, zf, channel_id)
                 except Exception as e:
-                    logging.error("Publishing error: {}".format(str(e)))
+                    logging.error("Error while publishing channel `{}`: {}".format(ccnode.get_channel_id(), str(e)))
```

Comment:
Any reason not to reuse `channel_id` here?

Reply:
Ooh, no reason I can see.  I'll change that.